integration/common.go

```diff
@@ -136,6 +153,74 @@ func CreateCredentialApplicationJWT(presentationDefinitionID, credentialJWT, man
 	return signed.String(), nil
 }
 
+func CreatePresentationDefinition() (string, error) {
+	logrus.Println("\n\nCreate our Presentation Definition:")
+	definitionJSON := getJSONFromFile("presentation-definition-input.json")
+	output, err := put(endpoint+version+"presentations/definitions", definitionJSON)
+	if err != nil {
+		return "", errors.Wrapf(err, "presentation definition endpoint with output: %s", output)
+	}
+
+	return output, nil
+}
+
+func ReviewSubmission(id string) (string, error) {
+	logrus.Println("\n\nCreate our review submission request:")
+	reviewJSON := getJSONFromFile("review-submission-input.json")
+	output, err := put(endpoint+version+"presentations/submissions/"+id+"/review", reviewJSON)
+	if err != nil {
+		return "", errors.Wrapf(err, "review submission endpoint with output: %s", output)
+	}
+
+	return output, nil
+}
+
+func CreateSubmission(definitionID, holderDID, holderPrivateKey, credentialJWT string) (string, error) {
+	logrus.Println("\n\nCreate our Submission:")
+	submissionJSON := getJSONFromFile("presentation-submission-input.json")
+
+	submissionJSON = strings.ReplaceAll(submissionJSON, "<DEFINITIONID>", definitionID)
```

Comment:
would really like us to switch to go templating -- https://pkg.go.dev/text/template

Reply:
Great idea. Will take that in a separate PR.

Reply:
ty